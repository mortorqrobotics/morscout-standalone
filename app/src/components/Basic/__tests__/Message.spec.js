// __tests__/Message.spec.js
import React from "react";
import renderer from "react-test-renderer";
import Message from "../Messages";

// console.log(require("canvas"));

test("renders correctly", () => {
  const tree = renderer
    .create(React.createElement(Message, {}, ["Hello"]))
    .toJSON();
  expect(tree).toMatchSnapshot();
});
