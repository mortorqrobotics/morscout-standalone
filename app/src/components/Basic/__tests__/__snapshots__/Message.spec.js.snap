// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message renders correctly 1`] = `
<div
  className="css-text-76zvg2"
  dir="auto"
>
  Hello
</div>
`;

exports[`Random Message renders correctly 1`] = `
<div
  className="css-text-76zvg2"
  dir="auto"
  onClick={[Function]}
>
  Hello
</div>
`;
